src/index.ts

```diff
@@ -11,6 +11,7 @@ import { auditLogger } from './logger';
 import { generateRedisConfig } from './lib/queue';
 
 const bypassSockets = !!process.env.BYPASS_SOCKETS;
+Error.stackTraceLimit = 50;
```

Comment:
might be reasonable (at some point, not necessarily here) to make this configurable via env var, that way we could pump it up in the cloud.gov web ui if we ever needed to